quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart Guide
+
+:information_source: Read this document before you start working on the [quickstart template](quickstart-template.md).
+
+## Introduction
+
+A quickstart introduces your users to your application for the first time.
+It focuses on the **primary feature** of the application and helps your users to start using the application as quickly as possible.
+
+A good quickstart answers the following questions:
+
+- Scope:
+   - What is the core purpose of this application?
+   - What is the minimal use case that is covered in this quickstart?
+- Install (if applicable):
+   - How do I download, install, and configure the application?
+   - How can I get any required access keys or authentication credentials?
+- Hello  World: 
+:memo: A hello world approach is the simplest and easiest task through which your user can get an end-to-end sense of how an application works. It also serves as a sanity check.
+   - How can I run a simple workflow for a core feature or a common use case?
+- Next steps:
+   - What other features are available to explore in the application?
+
+Quickstart guides are often confused with getting started and marketing guides. Though these documents help the users get acquainted with the application, they widely differ in their target audience. The following table describes the differences:
+
+| |Quickstart Guide|Getting Started Guide| Marketing Guide|
+|--------|----------------|----------------------|---------------|
+ |Target Audience|Domain experts who know the problem space|Beginners who are new to the problem space and the product|Business decision-makers who need to make strategic decisions such as whether to buy the product
+|Content|Minimal conceptual information|Detailed conceptual information on product/domain| No/less conceptual information; instead focuses on the benefits and the customers using the product
+|Focus|How-to| How-to and why|  Sales
+
+## Why do I need a quickstart?
+
+A quickstart is often the first opportunity for your users to form a positive impression on your product, and build an opinion on the technology it was built on. It can:
```

Comment:
```suggestion
A quickstart is often the first opportunity for your users to form a positive impression on your product, and form an opinion on the technology it was built on. It can:
```